config/deploy/production.rb

```diff
@@ -0,0 +1,61 @@
+# server-based syntax
+# ======================
+# Defines a single server with a list of roles and multiple properties.
+# You can define all roles on a single server, or split them:
+
+# server 'example.com', user: 'deploy', roles: %w{app db web}, my_property: :my_value
+# server 'example.com', user: 'deploy', roles: %w{app web}, other_property: :other_value
+# server 'db.example.com', user: 'deploy', roles: %w{db}
+server 'timeoverflow.org', user: 'deploy', roles: %w{app db web}
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.